develop/dev-guide-gui-mysql-workbench.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 使用 MySQL Workbench 连接 TiDB
+summary: 了解如何使用 MySQL Workbench 连接到 TiDB。
+---
+
+# 使用 MySQL Workbench 连接 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[MySQL Workbench](https://www.mysql.com/products/workbench/) 是为 MySQL 数据库用户提供的 GUI 工具集。
+
+> **警告**
+>
+> - 尽管由于 MySQL Workbench 兼容 MySQL，你可以使用 MySQL Workbench 连接到 TiDB，但 MySQL Workbench 并不完全支持 TiDB。由于 MySQL Workbench 将 TiDB 视为 MySQL，因此在使用过程中可能会遇到一些问题。
+> - 推荐使用其他官方支持 TiDB 的 GUI 工具，例如 DataGrip，DBeaver 以及 VS Code SQLTools。TiDB 完全支持的 GUI 工具的完整列表，参考 [TiDB 支持的第三方工具](/develop/dev-guide-third-party-support.md#gui)。
+
+在本文档中，你可以学习如何使用 MySQL Workbench 连接到 TiDB 集群。
+
+> **注意**
+>
+> 本文档适用于 TiDB Serverless、TiDB Dedicated 和本地部署的 TiDB。
+
+## 前置需求
+
+为了能够顺利完成本文中的操作，你需要：
+
+- [MySQL Workbench](https://dev.mysql.com/downloads/workbench/) **8.0.31** 或以上版本。
+- 准备一个 TiDB 集群。
+
+**如果你未有 TiDB 集群，可以按如下方式创建一个：**
+
+- （推荐） 参考 [使用 TiDB Serverless 构建 TiDB 集群](/develop/dev-guide-build-cluster-in-cloud.md) 来创建一个 TiDB Cloud 集群。
+- 参考 [部署本地测试集群](/quick-start-with-tidb.md#部署本地测试集群) 或 [部署生产 TiDB 集群](/production-deployment-using-tiup.md) 来创建一个本地集群。
+
+## 连接到 TiDB
+
+根据你选择的 TiDB 部署选项连接到 TiDB 集群。
+
+<SimpleTab>
+<div label="TiDB Serverless">
+
+1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，并进入集群的 Overview 页面。
+
+2. 点击右上角的 **Connect** 按钮，将会弹出连接对话框。
+
+3. 确认对话框中的配置和你的运行环境一致。
+
+    - **Endpoint Type** 选择 `Public`。
+    - **Branch** 选择 `main`。
+    - **Connect With** 选择 `MySQL Workbench`。
+    - **Operating System** 为你的运行环境。
+
+4. 点击 **Generate Password** 生成一个随机密码。
+
+    > **小贴士**
+    >
+    > 如果你之前已经生成过密码，可以直接使用原密码，或点击 **Reset Password** 重新生成密码。
+
+5. 启动 MySQL Workbench，并点击 **MySQL Connections** 标题附近的 **+**。
+
+    ![MySQL Workbench: add new connection](/media/develop/mysql-workbench-add-new-connection.png)
+
+6. 在 **Setup New Connection** 对话框中，配置以下连接参数：
+
+    - **Connection Name**：为该连接指定一个有意义的名称。
+    - **Hostname**：输入从 TiDB Cloud 连接对话框中的得到的 `HOST` 参数。
+    - **Port**：输入从 TiDB Cloud 连接对话框中的得到的 `PORT` 参数。
+    - **Username**：输入从 TiDB Cloud 连接对话框中的得到的 `USERNAME` 参数。
+    - **Password**：点击 **Store in Keychain ...**，输入 TiDB Serverless 集群的密码，然后点击 **OK** 保存密码。
```

Comment:
```suggestion
    - **Password**：点击 **Store in Keychain ...**  或 **Store in Vault**，输入 TiDB Serverless 集群的密码，然后点击 **OK** 保存密码。
```

Reply:
We can cover https://github.com/pingcap/docs/pull/16136 in this translation PR too😂